x-pack/plugins/canvas/storybook/main.ts

```diff
@@ -4,11 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { existsSync } = require('fs');
+const { join } = require('path');
+
+// Check for DLL
+if (!existsSync(join(__dirname, '../../../../built_assets/canvas_storybook_dll/manifest.json'))) {
+  // eslint-disable-next-line no-console
+  console.error(
+    'No DLL found. Run `node scripts/storybook --dll` from the Canvas plugin directory.'
+  );
+  process.exit(1);
+}
+
 module.exports = {
   stories: ['../**/*.stories.tsx'],
-  addons: [
-    '@storybook/addon-actions',
-    '@storybook/addon-knobs',
-    './storybook/addon/target/register',
-  ],
+  addons: ['@storybook/addon-actions', '@storybook/addon-knobs', './addon/src/register'],
```

Comment:
```suggestion
  addons: ['@storybook/addon-actions', '@storybook/addon-knobs', './addon/target/register'],
```

Reply:
`src` is incorrect, and will throw an error on startup.  The `scripts/storybook` script in `canvas` builds the addon.